lynx/build.gradle

```diff
@@ -3,10 +3,10 @@ apply plugin: 'checkstyle'
 
 android {
   compileSdkVersion 24
-  buildToolsVersion "24.0.0"
+  buildToolsVersion "24.0.1"
 
   defaultConfig {
-    minSdkVersion 8
+    minSdkVersion 10
```

Comment:
Remember to update the project ``README.md``. We've been supporting API 8 and now we need to specify that the min Api Level supported is 10.